README.md

```diff
@@ -3,7 +3,7 @@ Barstools
 
 ---
 
-![Test](https://github.com/freechipsproject/ucb-bar/barstools/Test/badge.svg)
+![CI Status](https://github.com/ucb-bar/barstools/actions/workflows/publish.yml/badge.svg)
```

Comment:
If we are publishing to Sonatype, we should also update the readme to mention what needs to be added to peoples `build.sbt` files.